src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -38,6 +38,10 @@ public abstract class FormulaType
 
         public static FormulaType OptionSetValue { get; } = new OptionSetValueType();
 
+        public static FormulaType Hyperlink { get; } = new HyperlinkType();
+
+        public static FormulaType Invalid { get; } = new InvalidType();
```

Comment:
Do these types participate in analysis? 
Eg, If X is a HyperLink, does Eval("If(..., x, ...)") return a hyperlink type?

Reply:
Yes.
These are being added to unblock IR Translation, which fails if it can't map the binding type to a FormulaType

Reply:
Where are they coming from? Does Dataverse register a hyperlink type?